<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test_process_to_user.TestProcessToUser-20200824070747" tests="1" file="test_process_to_user.py" time="0.001" timestamp="2020-08-24T07:07:47" failures="0" errors="1" skipped="0">
	<testcase classname="test_process_to_user.TestProcessToUser" name="test_process_to_user" time="0.001" timestamp="2020-08-24T07:07:47" file="spec/construct/test_process_to_user.py" line="8">
		<error type="UnicodeDecodeError" message="'utf-8' codec can't decode byte 0xac in position 1: invalid start byte"><![CDATA[Traceback (most recent call last):
  File "/home/travis/build/kaitai-io/ci_targets/tests/spec/construct/test_process_to_user.py", line 9, in test_process_to_user
    r = _schema.parse_file('src/process_rotate.bin')
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 309, in parse_file
    return self.parse_stream(f, **contextkw)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 300, in parse_stream
    return self._parsereport(stream, context, "(parsing)")
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 1981, in _parse
    subobj = sc._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 2439, in _parse
    return self.subcon._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 4496, in _parse
    return self.subcon._parsereport(io.BytesIO(data), context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 5534, in _parse
    return sc._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 1981, in _parse
    subobj = sc._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 2439, in _parse
    return self.subcon._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 698, in _parse
    return self._decode(obj, context, path)
  File "/home/travis/virtualenv/python3.8.1/lib/python3.8/site-packages/construct/core.py", line 1495, in _decode
    return obj.decode(self.encoding)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xac in position 1: invalid start byte
]]></error>
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
</testsuite>
